File: files/100-199/190/CH-190.R
library(tidyverse)
library(readxl)

path = "files/CH-190 Convert Text to number.xlsx"
input = read_excel(path, range = "B2:B5")
test  = read_excel(path, range = "D2:D5")

library(reticulate)
w2n = import("word2number")$w2n

result = input %>%
  mutate(Output = map_dbl(Date, ~w2n$word_to_num(.x))) %>%
  select(Output)

all.equal(result, test, check.attributes = FALSE)
#> [1] TRUE

File: files/100-199/190/CH-190.py
import pandas as pd
from word2number import w2n

path = "CH-190 Convert Text to number.xlsx"
input = pd.read_excel(path, usecols="B", skiprows=1, nrows=4)
test = pd.read_excel(path, usecols="D", skiprows=1, nrows=4)

input['Output'] = input.pop('Date').apply(w2n.word_to_num)

print(input.equals(test)) # True